src/pybind/mgr/dashboard/ci/cephadm/bootstrap-cluster.sh

```diff
@@ -8,7 +8,13 @@ chmod +x /root/bin/cephadm
 mkdir -p /etc/ceph
 mon_ip=$(ifconfig eth0  | grep 'inet ' | awk '{ print $2}')
 
-cephadm bootstrap --mon-ip $mon_ip --initial-dashboard-password {{ admin_password }} --allow-fqdn-hostname --skip-monitoring-stack --dashboard-password-noupdate --shared_ceph_folder /mnt/{{ ceph_dev_folder }}
+bootstrap_extra_options='--allow-fqdn-hostname --dashboard-password-noupdate'
+bootstrap_extra_options_not_expanded='--skip-monitoring-stack'
+{% if expanded_cluster is not defined %}
+  bootstrap_extra_options+=" ${bootstrap_extra_options_not_expanded}"
+{% endif %}
```

Comment:
I'm a bit lost here... Is this bash?

Reply:
It's a jinja conditional construct as  this script is to be invoked as part of `ceph_cluster.yml`.
See https://kcli.readthedocs.io/en/latest/index.html#overriding-parameters